src/test/java/uk/co/ramp/event/ContactEventProcessorTest.java

```diff
@@ -93,6 +97,95 @@ public void evaluateContact() {
     Assert.assertEquals(infectionEvent.nextStatus(), EXPOSED);
   }
 
+  @Test
+  public void evaluateContactCompliant() {
+
+    int infector = 0;
+    int infectee = 1;
+    int time = 0;
+
+    IsolationPolicy isolationPolicy = Mockito.mock(IsolationPolicy.class);
+    when(isolationPolicy.isContactIsolated(any(), any(), anyDouble(), anyDouble(), anyInt()))
+        .thenReturn(true);
+
+    ReflectionTestUtils.setField(eventProcessor, "isolationPolicy", isolationPolicy);
+
+    Case mock0 = mock(Case.class);
+    when(mock0.virusStatus()).thenReturn(SYMPTOMATIC);
+    when(mock0.isInfectious()).thenReturn(true);
+    when(mock0.isolationCompliance()).thenReturn(-0.5);
```

Comment:
Should the expected range of compliance values be between 0 and 1?

Reply:
The -ve is just to ensure the conditional fails. It saved mocking the sampler, this will be deleted anyway.